ChangeLog

```diff
@@ -64,6 +64,11 @@ Release date: TBA
 
   Closes #1555
 
+* Fix crash (in unreleased development) in ``no-member`` check when inspecting C
+  extension modules with ``extension-pkg-allow-list``.
+
+  Closes #6221
+
```

Comment:
```suggestion
```
I think we could add ``Closes #6221`` to the original log for the false negative (#1555 / #5544)